addons/common/fnc_isAttenuated.sqf

```diff
@@ -0,0 +1,96 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_isAttenuated
+
+Description:
+    Checks whether a unit is attenuated.
```

Comment:
Give explanation of what that means, eg:
`Checks whether a unit is attenuated, meaning in another compartment outside of outside voice range.`
Someone give a better one, I don't know how to describe that.